cmd/buildah/manifest.go

```diff
@@ -194,6 +194,8 @@ func init() {
 	flags.StringVar(&manifestPushOpts.certDir, "cert-dir", "", "use certificates at the specified path to access the registry")
 	flags.StringVar(&manifestPushOpts.creds, "creds", "", "use `[username[:password]]` for accessing the registry")
 	flags.StringVar(&manifestPushOpts.digestfile, "digestfile", "", "after copying the image, write the digest of the resulting digest to the file")
+	flags.BoolVarP(&manifestPushOpts.removeSignatures, "remove-signatures", "", false, "don't copy signatures when pushing image")
```

Comment:
removesignatures seems like a bad name for this?  Could it be copy-signatures and default to true?

Reply:
`--remove-signatures` and `--sign-by` are what `skopeo copy` calls these options.  Are you sure?

Reply:
I don't mind the --signby, but --remove-signatures does not really describe what is happening, I think --nosigs would make more sense and easier to type.  --remove-signatures sounds like signatures are being deleted,  it is also way to many characters. I would rather pick something new and then move skopeo to use the new name and alias it to --remove-signatures.

Reply:
I don't mind the --signby, but --remove-signatures does not really describe what is happening, I think --nosigs would make more sense and easier to type.  --remove-signatures sounds like signatures are being deleted,  it is also way to many characters. I would rather pick something new and then move skopeo to use the new name and alias it to --remove-signatures.